webapp/src/css/material.less

```diff
@@ -31,6 +31,11 @@
     div {
       flex-grow: 100;
     }
+    &.age {
+      span.relative-date.future {
+        color: red;
```

Comment:
I think it would be better to use one of the less variables in variables.less. That way we keep a consistent colour palette throughout the app. I would recommend `color: @error-color` which we use in other places.